tests/flytekit/unit/core/test_node_creation.py

```diff
@@ -452,3 +452,16 @@ def my_wf(a: str) -> str:
             return t1(a=a).with_overrides(task_config=None)
 
         my_wf()
+
+
+def test_override_image():
+    @task
+    def bar():
+        print("hello")
+
+    @workflow
+    def wf() -> str:
+        bar().with_overrides(container_image="hello/world")
```

Comment:
because people are used to
```
@task(container="{{ blah.fqn }}:{{ default.version }}")
   ...
```
can we make
```
bar().with_overrides(container_image="{{ blah.fqn }}:{{ default.version }}")
```

Reply:
both imageSpec and that work for me.